test/non_existent_command.bats

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bats
+
+load test_helpers
+
+setup() {
+  setup_asdf_dir
+  install_dummy_plugin
+  install_dummy_legacy_plugin
+  run asdf install dummy 1.0
+  run asdf install dummy 1.1
```

Comment:
I think these 4 lines can be removed. They should not be needed for the code your test exercises.

Reply:
Yep, local tests pass without those lines.